jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/bars/AppBarsTests.java

```diff
@@ -69,6 +71,18 @@ public void fadeImageOnScrollBarTest() {
         imageBar.has().noImageFadingOnScroll();
     }
 
+    @Test(description = "Test checks if app bar has image threshold or not")
+    public void thresholdBarTest() {
+        thresholdBar.show();
+        thresholdBar.has().imageFadingOnScroll();
+        thresholdBar.scrollBarToBottom();
+        thresholdBar.image().has().attr("style", "opacity: 0;");
+        thresholdBar.is().onBottomPosition();
+        thresholdBar.scrollBarToTop();
+        thresholdBar.image().has().attr("style", "opacity: 1;");
+        thresholdBar.is().onTopPosition();
+    }
+
```

Comment:
Sorry, only today found that it works only together 
![image](https://user-images.githubusercontent.com/55189259/214377693-476deb3a-d46a-4bd9-9025-baeb25d87c9b.png)
Let's discussm :) it will need changes on test-site